rust/main/hyperlane-base/src/settings/trace/mod.rs

```diff
@@ -75,12 +75,10 @@ impl TracingConfig {
                 .with_target("tower", Level::Info)
                 .with_target("tendermint", Level::Info)
                 .with_target("tokio", Level::Debug)
-                .with_target("tokio_util", Level::Debug)
```

Comment:
Is unsilencing this intentional?

Reply:
We may need this line if we want to increase the log level beyond Debug, I think.